### PR TITLE
Syntribos Output Improvemnets

### PR DESCRIPTION
**Pull request by kirankumbhar**
- Get HTTP response in output under failure instance
- Get failed strings in output for fuzz tests like xss, buffer overflow, sql injection, xml injection etc.

### CODE REVIEW

File: syntribos/issue.py [hunk 1]
```diff
@@ -42,7 +42,7 @@ class Issue(object):
 
     def __init__(self, defect_type, severity, description, confidence,
                  request=None, response=None, impacted_parameter=None,
-                 init_signals=[], test_signals=[], diff_signals=[]):
+                 init_signals=[], test_signals=[], diff_signals=[], failed_strings=[]):
```
Comment (subho007) [inline syntribos/issue.py]: Please set the default to `None` and check if it is not `None` then initialise the array. It is a bad idea to have `[]` as a default args.

File: syntribos/tests/fuzz/base_fuzz.py [hunk 2]
```diff
@@ -169,7 +169,7 @@ def extend_class(cls, new_name, fuzz_string, param_path, kwargs):
         new_cls.param_path = param_path
         return new_cls
 
-    def register_issue(self, defect_type, severity, confidence, description):
+    def register_issue(self, defect_type, severity, confidence, description, failed_strings=[]):
```
Comment (subho007) [inline syntribos/tests/fuzz/base_fuzz.py]: same here, initialise it with `None`

**Review by subho007 — CHANGES_REQUESTED** [review]
Check out my comments regarding having `[]` as default

File: syntribos/issue.py (hunk 1 above)
Comment (kirankumbhar) [reply syntribos/issue.py]: Done 👍🏼

File: syntribos/tests/fuzz/base_fuzz.py (hunk 2 above)
Comment (kirankumbhar) [reply syntribos/tests/fuzz/base_fuzz.py]: Done 👍🏼

**Review by subho007 — APPROVED** [review]
LGTM 👍

**Pull request merged**